src/main/java/com/shopify/model/ShopifyCustomCollectionCreationRequest.java

```diff
@@ -0,0 +1,54 @@
+package com.shopify.model;
+
+public class ShopifyCustomCollectionCreationRequest {
+
+	private final ShopifyCustomCollection request;
+
+
+	public static interface InitialValueStep {
+		HandleStep withTitle(final String title);
+	}
+
+	public static interface HandleStep {
+		BuildStep withHandle(final String handle);
+	}
+
+	public static interface BuildStep {
+		ShopifyCustomCollectionCreationRequest build();
+	}
+
+
+	public static InitialValueStep newBuilder() {
+		return new Steps();
+	}
+
+	public ShopifyCustomCollection getRequest() {
+		return request;
+	}
+
+	private ShopifyCustomCollectionCreationRequest(final ShopifyCustomCollection request) {
+		this.request = request;
+	}
+
+	private static class Steps implements InitialValueStep, HandleStep, BuildStep {
+
+		private final ShopifyCustomCollection request = new ShopifyCustomCollection();
+
+		@Override
+		public ShopifyCustomCollectionCreationRequest build() {
+			return new ShopifyCustomCollectionCreationRequest(request);
+		}
+
+		@Override
+		public HandleStep withTitle(final String title) {
+			this.request.setTitle(title);
+			return this;
+		}
+
+		@Override
+		public BuildStep withHandle(final String handle) {
```

Comment:
This is not required. You should include an optionals step so that you don't need to provide this field or any fields that aren't required if you don't want to. To see an optionals step, please review the model:
ShopifyGetCustomersRequest

Reply:
Thanks for the example! It's fixed now and also missing field are introduced to builder